### PR TITLE
Post delete

### PR DESCRIPTION
**Pull request by BobZcao**
create product method created

### CODE REVIEW

**Comment by BobZcao** [comment]
I did not include the delete part because @omersolmazer is doing that part

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<component name="InspectionProjectProfileManager">
```
Comment (p2hang) [inline .idea/inspectionProfiles/profiles_settings.xml:1]: Please remove these intellij files.
I think they should have already been ignored in .gitignore

File: server.py [hunk 2]
```diff
@@ -41,3 +41,27 @@ def update_product(id):
...
+
+@app.route('/inventory/products/', methods= ['POST'])
+def create_products():
+    """
+
+    This method will create a storage for a new product
+
+    Args:
+
+    Returns:
```
Comment (p2hang) [inline server.py:55]: I think we may also return the product id for the product created?
but looks good!

**Review by p2hang — CHANGES_REQUESTED** [review]
Looks good!
1. But please remove the intellij files(.xml, .iml).

2. And for the create product method I think we probably can also return the ID of the product created.(no changes requested here, depends on what you think)

**Pull request closed without merging**